### PR TITLE
Update azure-cns image to v1.4.0

### PR DESCRIPTION
**Pull request by mattstam**
Updating CNS to v1.4.0 for fixing a connection reset issue that was found.

### CODE REVIEW

**Comment by mattstam** [comment]
@paulgmiller

File: vhdbuilder/packer/components.json [hunk 1]
```diff
@@ -74,7 +74,8 @@
       "downloadURL": "mcr.microsoft.com/containernetworking/azure-cns:*",
       "versions": [
         "v1.2.9",
```
Comment (paulgmiller) [inline vhdbuilder/packer/components.json:76]: rip out 1.2.9? 1.3.3's been out a while

**Comment by paulgmiller** [comment]
I still think there are unittests for components.json?

**Review by paulgmiller — APPROVED** [review]

File: vhdbuilder/packer/components.json (hunk 1 above)
Comment (mattstam) [reply vhdbuilder/packer/components.json]: removed!

**Pull request merged**